DemoEdgeApp/ModuleWrapper/TaskTimer.cs

```diff
@@ -0,0 +1,72 @@
+namespace ModuleWrapper
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Serilog;
+
+    public class TaskTimer
+    {
+        CancellationTokenSource cancellationTokenSource;
+        TimeSpan timerPeriod;
+        Action onElapsedCallback;
+        bool continueOnError;
+
+        public TaskTimer(Action onElapsedCallback,
+            TimeSpan timerPeriod,
+            CancellationTokenSource cancellationTokenSource,
+            bool continueOnError = true)
+        {
+            this.cancellationTokenSource = cancellationTokenSource;
+            this.timerPeriod = timerPeriod;
+            this.onElapsedCallback = onElapsedCallback;
+            this.continueOnError = continueOnError;
+        }
+
+        public void Start()
+        {
+            Task elapsedTask = null;
+            elapsedTask = new Task((x) =>
```

Comment:
typically use _ when a lambda parameter is ignored instead of 
`new Task((x) =>`
use
`new Task((_) =>`

Reply:
fixed